lib/ffaker/bank.rb

```diff
@@ -103,6 +103,18 @@ def card_type
       fetch_sample(CARD_TYPES)
     end
 
+    def loan_interest_rate(min_rate = 1.5, max_rate = 15.0)
+      rand(min_rate..max_rate).round(2)
```

Comment:
Use fetch_sample please.
```suggestion
      fetch_sample(min_rate..max_rate).round(2)
```